cargo-apk/src/main.rs

```diff
@@ -9,6 +9,7 @@ fn main() -> anyhow::Result<()> {
     let builder = ApkBuilder::from_subcommand(&cmd)?;
 
     match cmd.cmd() {
+        "check" => builder.check()?,
         "build" => {
             for artifact in cmd.artifacts() {
```

Comment:
Seeing that `build` handles artifacts, should `check` also do something like that so that `-p` and `--example` work, or is that implicitly working already when `self.cmd.args()` is passed to `cargo check`?